.github/workflows/ci.yml

```diff
@@ -206,6 +216,9 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile --cache-folder $(yarn cache dir)
 
+      - name: Generate open-api
+        run: yarn codegen:socket
+
```

Comment:
Doesn't `install` job do this already?

Reply:
No I removed postinstall

Reply:
Correct, but the `install` job is required by `build` and `deploy` needs `build`, so this seems unnecessary

Reply:
`codegen:socket` was run as post-install and it was triggered automatically once install finishes. 
So earlier it was like install => post-install => build => deploy
post-install will be called after install. 

Now I removed `post-install` and calling it directly as `yarn codegen:socket` .  I didn't understand what is unnecessary?

https://yarnpkg.com/advanced/lifecycle-scripts

Reply:
Yes you are correct. I didn't see where you commented. Will remove it in next PR. I added here to fix `deploy` and then forget to remove when I added at top.

Reply:
Sorry again mis read the line number when its folded in github. It was added in Synpress. `Synpress` is not using `install`.